bundles/org.openhab.binding.vdr/README.md

```diff
@@ -0,0 +1,116 @@
+
+To configure a VDR Hostname or IP Address and the actual SVDRP Port are required. Please note that until VDR version 1.7.15 the standard SVDRP port was 2001 and after that version it changed to 6419. The VDR configuration file svdrphosts.conf needs to be configured to allow SVDRP access from host where openHAB instance is running. Please check VDR documentation if you are unsure about this.
+
+| Configuration Parameter | Default          | Required | Description                                                  |
+|-------------------------|------------------|:--------:|--------------------------------------------------------------|
+| host                    |                  |   Yes    | Hostname or IP Address of VDR instance                       |
+| port                    | 6419             |   Yes    | SVDRP Port of VDR instance                                   |
+| refresh                 | 30               |   No     | Interval in seconds the data from VDR instance is refreshed  |
+
+A typical thing configuration would look like this:
+
+```
+Thing vdr:vdr:livingRoom "VDR" @ "LivingRoom"	[ host="192.168.0.51", port=6419, refresh=30 ]
+```
+
+
+## Channels
+
+`power`, `channel` and `volume` can be used for basic control of your VDR. `diskUsage` might be used within a rule to notify if disk space for recordings runs short. It is also possible to display custom messages on VDR OSD, please use `message` for this. You can build your own remote control widget in openHAB by using the `keyCode` channel.
+
+Also you can show information about the current channel's program on your VDR by displaying the EPG Event Channels in your favorite openHAB user interface.
+
+To turn on the device VDR is running on please use Wake-On-LAN functionality from Network Binding.
+
+
+| channel              | type     | description                             |
+|----------------------|----------|-----------------------------------------|
+| power                | Switch   | Power State (to switch off VDR)         |
+| channel              | Number   | Current Channel Number (can be changed) |
+| channelName          | String   | Name of Current Channel                 |
+| volume               | Dimmer   | Current Volume                          |
+| recording            | Switch   | Is currently a Recording Active?        |
+| diskUsage            | Number   | Current Disk Usage in %                 |
+| message              | String   | Send Message to be displayed on VDR     |
+| keyCode              | String   | Send Key Code of Remote Control to VDR  |
+| currentEventTitle    | String   | Current EPG Event Title                 |
+| currentEventSubTitle | String   | Current EPG Event Sub Title             |
+| currentEventBegin    | DateTime | Current EPG Event Begin                 |
+| currentEventEnd      | DateTime | Current EPG Event End                   |
+| currentEventDuration | Number   | Current EPG Event Duration in Minutes   |
```

Comment:
You could make use of Units of Measure and make the type `Number:Time`. See https://www.openhab.org/docs/concepts/units-of-measurement.html

Same for `nextEventDuration`.